packages/faceted-search/src/components/BasicSearch/BasicSearch.scss

```diff
@@ -1,10 +1,17 @@
 .tc-basic-search {
 	align-items: center;
 	display: flex;
-	flex-wrap: wrap;
+	flex-direction: row;
 	height: inherit;
-	margin-right: $padding-large;
 	overflow: auto;
 	padding-left: $padding-large;
-	width: calc(100% - 15rem);
+	flex-grow: 1;
```

Comment:
I think you can remove that. Not sure that the search is in a flex container, and either way, this kind of behavior should be done in its block container.